js/data/move_data.js

```diff
@@ -355,6 +355,11 @@ var MOVES_RBY = {
 		bp: 0,
 		type: 'Grass'
 	},
+	'Struggle': {
+		bp: 1,
```

Comment:
Should be 50. Were you thinking of zp maybe?

Reply:
Ooh I was looking at pp lmao that makes a lot more sense, I was wondering why it was so low